src/documentation/components/ComponentExample.vue

```diff
@@ -86,20 +166,36 @@ export default class ComponentExample extends Vue {
   @Prop() title?: string;
   @Prop() titleSize?: 'h3' | 'h4';
   @Prop() tabs?: TabsInterface[];
+  @Prop() headTabs?: HeadTabsInterface[];
   @Prop() padding?: string;
   @Prop() additionalClasses?: string;
   @Prop() additionalStyle?: string;
 
   chiTabs: any;
+  chiHeadTabs: any;
 
   mounted() {
     const chiTabs = document.getElementById(
-      'code-snippet-tabs' + this.$props.id
+      'code-snippet-tabs-' + this.$props.id
     );
+    const chiHeadTabs = document.getElementById('head-tabs-' + this.$props.id);
+
+    if (chiTabs) this.chiTabs = chi.tab(chiTabs);
+    if (chiHeadTabs) {
+      this.headTabs?.forEach((tab: HeadTabsInterface) => {
+        const codeSnippetTab = document.getElementById(
+          `code-snippet-tabs-${this.$props.id}-${tab.id}`
+        );
+
+        if (codeSnippetTab) chi.tab(codeSnippetTab);
+      });
+
+      this.chiHeadTabs = chi.tab(chiHeadTabs);
+    };
+  }
 
-    if (chiTabs) {
-      this.chiTabs = chi.tab(chiTabs);
-    }
+  emitHeadTabsChange(tab: HeadTabsInterface) {
+    this.$emit(COMPONENT_EXAMPLE_EVENTS.CHI_HEAD_TABS_CHANGE, tab);
   }
```

Comment:
@josecarlosgonzalezv Previously I have created a seperate component componentTabExample to handle multiple tab scenarios. Those changes are currently in tabs component PR https://github.com/CenturyLink/Chi/pull/321. Did ypu get a chance to look into it and use it for this drawer component?